make.bat

```diff
@@ -0,0 +1,35 @@
+@ECHO OFF

```

Comment:
The sphinx website suggests that the ``sphinx-quickstart`` should create a GNU makefile as well - could you add this? GeNN does fully support Windows, but our build server (and my desktop machine) run Linux so this would be convenient